src/dbus/server/introspect.xml

```diff
@@ -246,8 +246,15 @@
       <arg name="epskc" type="s" direction="out"/>
     </method>
 
-    <!-- DeactivateEphemeralKeyMode: Deactivate ePSKc mode. -->
+    <!-- DeactivateEphemeralKeyMode: Deactivate ePSKc mode.
+      @retain_active_session: 
```

Comment:
could mention the default value is false?

Reply:
Thanks，accepted.